apps/api/v2/src/modules/auth/guards/organizations/is-admin-api-enabled.guard.ts

```diff
@@ -43,7 +43,7 @@ export class IsAdminAccessGuard implements CanActivate {
       const adminAPIAccessIsEnabledInOrg = await this.organizationsRepository.fetchOrgAdminApiStatus(
         Number(organizationId)
       );
-      if (!adminAPIAccessIsEnabledInOrg) {
+      if (!adminAPIAccessIsEnabledInOrg && !org?.isPlatform) {
```

Comment:
In case of org being a platform we would call `this.organizationsRepository.fetchOrgAdminApiStatus` without it actually being ever used.

Reply:
Oh yeah, let me sort it out.